docs/sdk/web/wordpress/README.mdx

```diff
@@ -0,0 +1,119 @@
+---
+slug: /sdk/wordpress
```

Comment:
Need to rebase master and put this doc in `/quick-starts/framework/wordpress` folder

Reply:
updated